checkbox-support/checkbox_support/parsers/udevadm.py

```diff
@@ -1517,7 +1517,7 @@ def run(self):
                     d.category = "VIDEO"
 
         for device in list(self.devices.values()):
-            if device.category == "HIDRAW" and device._stack:
+            if device.category == "HIDRAW" and len(device._stack) > 1:
                 for parent in (device._stack[-1], device._stack[-2]):
```

Comment:
See if you like this other solution more. I have no idea what that following if is doing so I would like to call it if we have at least 1 parent. 
```suggestion
            if device.category == "HIDRAW" and device._stack:
                for parent in reversed(device._stack[-2:]):
```

Reply:
I think I would like to use the author's decision of checking these two parents only when they exist